### DISCUSSION TITLE
[Android]Does react native iap support  PBL 5.2.1 or PBL 6.0.1 or higher.

### DISCUSSION THREAD

**Discussion by ghost** [hyochan/react-native-iap#2553] [Q&A] [answered]
according the google play integrate [document](https://developer.android.com/google/play/billing/integrate), 
`If your app is targeting Android 14 or higher, you must update to [PBL 5.2.1](https://developer.android.com/google/play/billing/release-notes#5-2-1) or [PBL 6.0.1](https://developer.android.com/google/play/billing/release-notes#6-0-1) or higher.`

Does the latest version of react-native-iap already support PBL 5.2.1 or PBL 6.0.1 or higher?

**Comment by bitfabrikken** [comment, marked as answer]
I think so.

This is in gradle.properties:
```
RNIap_playBillingSdkVersion=5.2.1
```

**Comment by scuti-dai** [comment]
I have a same question
 "react-native-iap": "^11.0.0-rc.7". This use version 5.0.0

But if i use version 12, it did support for android 14 and RNIap_playBillingSdkVersion=5.2.1

![Screenshot 2023-09-27 at 10 46 35](https://github.com/dooboolab-community/react-native-iap/assets/104118833/8f248283-75a0-418d-9d7c-e7453d544fe5)
